pix4d-dependabot/lib/helpers/path_level.rb

```diff
@@ -8,6 +8,15 @@ def select_path_per_directory(git_tree, directory)
   files.map { |item| item["path"] }
 end
 
+def append_to_path(path)
```

Comment:
```suggestion
def normalize(path)
```
or
```suggestion
def normalize_path(path)
```
but definitely not `APPEND_to_path` at best it will _prepend_ a slash and _remove_ a trailing slash.

Reply:
Done as suggested.